setup.py

```diff
@@ -8,7 +8,16 @@
 with open("README.md") as readme_file:
     readme = readme_file.read()
 
-requirements = ["Click>=6.0", "python-jose==3.0.1", "flask==1.0.2", "requests==2.20.1", "PyYAML==5.1"]
+requirements = [
+    "Click>=6.0",
+    "python-jose==3.0.1",
+    "flask==1.0.2",
+    "requests==2.20.1",
+    "PyYAML==5.1",
+    "python-dateutil",
+    "requests-cache==0.5.0",
+    "pyxdg==0.26",
+]
```

Comment:
Not necessary, but it might be nice to move to requirements.txt and then read the file in as:

```
with open('requirements.txt') as f:
  requirements=f.read().splitlines()
```